monolithic/pom.xml

```diff
@@ -9,7 +9,6 @@
         <version>0.0.1-SNAPSHOT</version>
     </parent>
 
-    <groupId>com.epam.microservices</groupId>
```

Comment:
the same question. why `monolithic` project is excluded from the group?

Reply:
Just because it is the same as in the parent `pom.xml` so that it is redundant.